main/src/cgeo/geocaching/storage/DataStore.java

```diff
@@ -2243,6 +2255,7 @@ public static boolean storeIntoDatabase(final Geocache cache) {
         values.put("preventWaypointsFromNote", cache.isPreventWaypointsFromNote() ? 1 : 0);
         values.put("owner_guid", cache.getOwnerGuid());
         values.put("emoji", cache.getAssignedEmoji());
+        values.put("alcMode", cache.getAlcMode());
```

Comment:
This line is fine, but you are missing to read this value on loading the cache, see method `createCacheFromDatabaseContent`, where you should add a line `cache.setAlcMode(cursor.getInt(46));` - this seems to be the culprit why the mode value gets lost on loading the cache from database.

Reply:
Thx much ... will go after this tomorrow

Reply:
Done